docker/build_scripts/build_utils.sh

```diff
@@ -117,6 +100,32 @@ function do_openssl_build {
 }
 
 
+function _fetch_source {
```

Comment:
May be worth documenting the method. Also what about the following signature:

```
_fetch_source <url> <dest> [--method (curl|get)]
```

The `file` would be obtained from the url using `file=$(basename ${url})`

Reply:
Yeah, I might make another pass at this to pretty things up and reduce duplication after we get the immediate issue fixed.